libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.ts

```diff
@@ -562,9 +580,7 @@ export class SkyTextEditorComponent
         this.labelText ? this.errorId : '',
         this.ngControl.errors,
       );
-      this.#adapterService.setRequiredAttribute(
-        SkyFormsUtility.hasRequiredValidation(this.ngControl),
-      );
+      this.#adapterService.setRequiredAttribute(this.isEditorRequired);
```

Comment:
Why are we moving away from the shared utility here? It seems like we should still use the utility but just set a protected field. 

The utilities issues were purely due to date-range-picker's setup on the other PR. I would like to standardize to using when we can. We do this in a few components already. 

The utility would also allow us to not add the input on `required` as it handles template forms in the standard case - again - that was purely a date range picker issue that caused it to not work there.